### PR TITLE
Documentation

### LINKED ISSUE #17: Documentation On Endpoints

### PR DESCRIPTION
**Pull request by jordy1611**
### Project Side:
- [ ] FE
- [x] BE
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
Adds a bare bones landing page when visiting the base url for the api.
Very bare bones because flask deploy is not cooperative with css and setting of javascript.

Resolves #17

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ coverage*
 .DS_Store
 .env
 /env
+.coveragerc
```
Comment (kathytbui) [inline .gitignore:10]: This should have been ignored, but it's showing up in this PR. Should we delete it?

**Review by kathytbui — APPROVED** [review]
Thanks for setting up the html files for the docs!

File: .gitignore (hunk 1 above)
Comment (jKwibe) [reply .gitignore:10]: There is a merge conflict that needs to be resolved first @jordy1611 @Kathybui732

Comment (jKwibe) [reply .gitignore:10]: Why is the CI failing?

**Comment by jKwibe** [comment]
There is a merge conflict that needs to be resolved first @jordy1611 @Kathybui732

**Comment by jKwibe** [comment]
@jordy1611 I think you should pull the main brach to this branch, fix the conflicts the push it.

**Pull request merged**